### ISSUE TITLE
"no zip file content passed in" when deploying via Metadata API on node v0.12.x

### ISSUE THREAD

**Issue by joeferraro** [jsforce/jsforce#241]
when running on v0.10.x all operations complete successfully

**Comment by joeferraro** [comment]
fwiw changing https://github.com/jsforce/jsforce/blob/master/lib/api/metadata.js#L488 to:

```
zipInput.on('data', function(data) {
  bufs.push(data);
});
```

resolves the issue in node 0.10.x and 0.12.x, however, i'm not entirely clear why at the moment ...

**Comment by jeancdc** [comment]
Can you detail what we must to do to fix this problem?

**Comment by stomita** [comment]
Reproduced in our test suite. Will be fixed in master soon.

**Comment by jeancdc** [comment]
Sorry, I didn't understood...

**Comment by stomita** [comment]
@jeancdc if you need quickfix you can change the line of source code as @joeferraro suggested. If you would not like to, use Node.js 0.10.x or please wait till the official fix and support of 0.12.x.

**Comment by stomita** [comment]
Updated the fix in master, will be included ver. 1.5.
